src/lib/messenger.js

```diff
@@ -23,6 +23,19 @@ export default function messenger(bot) {
       const text = message.text.toLowerCase();
       const user = message.user || {};
 
+      function stripText(input) {
+        let commandText = '';
+        let command = '';
+
+        if (input.indexOf(' ') !== -1 && input.startsWith('/')) {
```

Comment:
There could be a separate test for the behavior of providing the parameter undefined to the function, in order to specify how it should behave. Then it could probably just be sovled by adding a default value to the parameter in the function signature.

I would say it's proper to at least think about it, it might be as you say, that it doesn't really matter, and if it throws an error it's just good, but I say only as long as you have thought about it :)